nav2_msgs/action/Empty.action

```diff
@@ -0,0 +1,7 @@
+#goal definition
```

Comment:
I don't think we need this action. Also, "Empty" isn't really an action, but a description of the messages used. Better to use a descriptive name for the action.

Reply:
Since `Stop` is going away I'll delete it.